Microsoft.HealthVault.Fhir/Units/UnitResolver.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Microsoft.HealthVault.Fhir.Units
+{
+    internal class UnitResolver
+    {
+        private static UnitResolver _instance;
+
+        public List<UnitConversion> UnitConversions { get; set; }
+
+        private UnitResolver()
+        {
+            UnitConversions = JsonConvert.DeserializeObject<List<UnitConversion>>(File.ReadAllText(@"Data\UnitConversions.json"));
+        }
+
+        public static UnitResolver Instance
```

Comment:
Agree, please implement locks on the instance to ensure it is thread safe. 

---
In reply to: [127044951](https://github.com/Microsoft/healthvault-fhir-library/pull/7#discussion_r127044951) [](ancestors = 127044951)